dev/InfoBar/InfoBar.idl

```diff
@@ -0,0 +1,143 @@
+﻿namespace MU_XC_NAMESPACE
+{
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum InfoBarCloseReason
+{
+    CloseButton = 0,
+    Programmatic = 1
+};
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum InfoBarSeverity
```

Comment:
This can be used in a variety of different scenarios. It should probably be generalized to just 'Severity'

Reply:
I understand where you're coming front, but we keep names control-specific.